packages/liveview/src/element.rs

```diff
@@ -119,6 +119,12 @@ impl RenderedElementBacking for LiveviewElement {
     }
 }
 
+impl HasResizeData for LiveviewElement {
```

Comment:
Do we use this implementation? The default methods for HasResizedData just return an error, so I don't know if this is useful to keep

Reply:
Ok.